test/postgres.test.js

```diff
@@ -448,3 +449,108 @@ describe('postgres store API V2.0.0', function () {
     })
   })
 })
+
+var siDefault = Seneca({
```

Comment:
the top part of the tests is a duplicate of the seneca store 1.1 plugin version, those will be removed